NLCP/Components/PersonComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using NLCP.Constants;
+using NLCP.Entities;
+using NLCP.Interfaces;
+
+namespace NLCP.Components
+{
+    // for now we can have this class but we might want to split it into locomotion, user input response and so on
+    public class PersonComponent : DrawableGameComponent
+    {
+        private Sprite _sprite;
+
+        public PersonComponent(Game game) : base(game)
+        {
+        }
+
+        protected override void LoadContent()
+        {
+            base.LoadContent();
+
+            var assetName = $"textures/people/{PeopleTextureNames.Placeholder}/{PersonTextureNames.MovementHorizontal}0"; // load frame zero of the horizontal movement
```

Comment:
If we are going to use a consistent or standard name formatting for the animation frames It may be good to set up a helper or something for building the path for us - I think for now this looks good though and we can evaluate what makes sense as we flesh it out more and have a better idea of what some of the non-placeholder type paths might look like

Reply:
Good point, I have created a card in codecks for this